JSONAPI.TodoMVC.API/Controllers/TodosController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using JSONAPI.Core;
+using JSONAPI.EntityFramework;
+using JSONAPI.EntityFramework.Http;
+using JSONAPI.TodoMVC.API.Models;
+
+namespace JSONAPI.TodoMVC.API.Controllers
+{
+    public class TodosController : ApiController<Todo, TodoMvcContext>
+    {
+        protected readonly TodoMvcContext DbContext;
+
+        public TodosController(TodoMvcContext dbContext)
+        {
+            DbContext = dbContext;
+        }
+
+        protected override IMaterializer MaterializerFactory()
+        {
+            return new EntityFrameworkMaterializer(DbContext);
+        }
+    }
```

Comment:
Right or wrong, it really bugs me that this class has any content, because everything in here is for testing purposes. Same for a lot of the stuff in Startup too, I think...much code exists strictly to allow constructor injection for tests and has no other purpose. At the very least, I'll ask the question, is there something we can add to `ApiController` or `IMaterializer` instead that would facilitate testing and allow this cruft to be omitted?